flow/environments/umn.py

```diff
@@ -0,0 +1,15 @@
+"""Environment for the Minnesota Supercomputing Institute at University of Minnesota"""
+from ..environment import DefaultTorqueEnvironment
+
+
+class MSIEnvironment(DefaultTorqueEnvironment):
```

Comment:
This template should match a particular cluster (Mangi). Other clusters (e.g. Mesabi) administered by the Minnesota Supercomputing Institute should have their own classes, because scheduling options will be unique to the cluster.
```suggestion
class MangiEnvironment(DefaultTorqueEnvironment):
```